todo/src/reducers/index.js

```diff
@@ -0,0 +1,41 @@
+const initialState = {
+    tasks: [
+        {
+            id: 'startertask',
+            text: 'Starter Task',
+            completed: false
+        }
+    ]
+}
+
+export default (state = initialState, action) => {
```

Comment:
Destructuring type and payload off of action here would do a good bit to reducing some of the repetition here.